tests/root_io/param_reading_rdataframe.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+
+import sys
+import ROOT
+
+
+ROOT.gInterpreter.Declare("#include <podio/utilities/RootHelpers.h>")
+df = ROOT.RDataFrame("events", sys.argv[1])
+
+evtWeights = (
+    df.Define(
+        "params",
+        "podio::root_utils::loadParamsFrom(GPIntKeys, GPIntValues, GPFloatKeys, GPFloatValues, GPDoubleKeys, GPDoubleValues, GPStringKeys, GPStringValues)",
+    )
```

Comment:
@Zehvogel, does this look like an acceptable compromise for using GenericParameters in RDataFrame until we have a proper RDataSource?

Reply:
yes thx!